### PR TITLE
fix(technical integration): role upload findings

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Fixed help link, funky character display and the success message display in user role upload

## Why

To display valid/proper data

## Issue

Ref: CPLP-3473

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by nidhigarg-bmw — APPROVED** [review]
lgtm

**Review by manojava-gk — APPROVED** [review]
lftm

File: src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ export default function TechnicalIntegration() {
             setUploadCSVError(true)
           }
         }
-        reader.readAsText(file)
+        reader.readAsText(file, 'ISO-8859-4')
```
Comment (oyo) [inline src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx:251]: ISO-8859-4 is a specific encoding for north european countries (https://en.wikipedia.org/wiki/ISO/IEC_8859-4) - our files should generally be in unicode, specifically utf-8

Does `reader.readAsText(file, 'utf-8')` not work?

If it doesn't work then the file is probably in an unsupported encoding and the user should make sure to only upload files that are in supported formats.

**Review by oyo — CHANGES_REQUESTED** [review]
question about uncommon character encoding.

File: src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx [hunk 2]
```diff
@@ -99,6 +100,39 @@ export default function TechnicalIntegration() {
   const [techUserProfiles, setTechUserProfiles] = useState<string[]>([])
   const [enableUserProfilesErrorMessage, setEnableUserProfilesErrorMessage] =
     useState(false)
+  const [selectedEncoding, setSelectedEncoding] = useState<string>('utf-8')
+  const [uploadFileInfo, setUploadFileInfo] = useState<DropzoneFile[]>([])
+  const unicodeSelectItems = [
+    {
+      id: 1,
+      title: 'utf-8',
+      value: 'utf-8',
```
Comment (oyo) [inline src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx:109]: Hey, doesn't it understand uppercase 'UTF-8'? - I think it should also work and looks nicer if all are upper case

**Review by oyo — CHANGES_REQUESTED** [review]
one tiny remark

**Review by oyo — APPROVED** [review]

**Comment by jjeroch** [comment]
rollback needed - issue: 

- wrong branch
- wrong changeLog update

@oyo  as we spoke - please make sure that 2 level pr check is executed - ping me if you need a quick check instead of merging with a single review

**Pull request merged**